exercises/isbn-verifier/description.md

```diff
@@ -9,27 +12,35 @@ The program should allow for ISBN-10 without the separating dashes to be verifie
 
 ## ISBN
 
-Let's take a random ISBN-10 number, say `3-598-21508-8` for this.
+The ISBN-10 format is 9 digits (0 to 9) plus one check character (either a digit or an X only). In the case the check character is an X, this represents the value '10'. These may be communicated with or without hyphens, and can be checked for their validity by the following formula:
+
+```
+(x1 * 10 + x2 * 9 + x3 * 8 + x4 * 7 + x5 * 6 + x6 * 5 + x7 * 4 + x8 * 3 + x9 * 2 + x10 * 1) mod 11 == 0
+```
+
+If the result is 0, then it is a valid ISBN-10, otherwise it is invalid.
+
+## Example
+
+Let's take a random ISBN-10 number, for instance `3-598-21508-8`.
 The first digit block indicates the group where the ISBN belongs. Groups can consist of shared languages, geographic regions or countries. The leading '3' signals this ISBN is from a german speaking country.
 The following number block is to identify the publisher. Since this is a three digit publisher number there is a 5 digit title number for this book.
 The last digit in the ISBN is the check digit which is used to detect read errors.
```

Comment:
All this stuff about countries and publishers doesn't belong in the example.

Reply:
Again, I agree but was not confident in stripping it out without another opinion. If you think this should be removed then I'll do that too! :slightly_smiling_face:

Reply:
The actual details of each section aren't relevant to the problem so removing them completely is not a bad option.

Another option is moving them up into the "about ISBN" section if we want to do more than just refer people to Wikipedia.